### PR TITLE
Fix secopts types

### PR DESCRIPTION
**Pull request by runcom**
Fix https://github.com/docker/docker/issues/28113

@vikstrous PTAL

### CODE REVIEW

File: client/client.go [hunk 1]
```diff
@@ -58,7 +58,7 @@ import (
 )
 
 // DefaultVersion is the version of the current stable API
-const DefaultVersion string = "1.23"
+const DefaultVersion string = "1.25"
```
Comment (thaJeztah) [inline client/client.go]: We should probably put a comment here, and in https://github.com/docker/docker/blob/70459d6f1ec98fa0aa0bf0d873838a9c4b5d3236/api/common.go#L24, that both should be kept in sync

Also, wondering why bumping these versions isn't described in https://github.com/docker/docker/blob/master/project/RELEASE-CHECKLIST.md#22-update-api-version-on-master

Comment (runcom) [inline client/client.go]: > Also, wondering why bumping these versions isn't described in https://github.com/docker/docker/blob/master/project/RELEASE-CHECKLIST.md#22-update-api-version-on-master

great question, I don't know

> We should probably put a comment here, and in https://github.com/docker/docker/blob/70459d6f1ec98fa0aa0bf0d873838a9c4b5d3236/api/common.go#L24, that both should be kept in sync

ack

Comment (vdemeester) [inline client/client.go]: I think it's because it was part of `engine-api` ?

Comment (runcom) [inline client/client.go]: should we drop one and just use the other instead? (I would remove the `client/client.go` one)

Comment (thaJeztah) [inline client/client.go]: Yes, I was thinking of that, and wondering why we didn't do that. Is it ok in this case to import one into the other? (possibly these packages were meant to be kept separate)

**Comment by vikstrous** [comment]
I don't know this code, but if it does what it claims to do I'm happy. :+1:

**Comment by runcom** [comment]
@vikstrous can you test if this patch fixes your issue though?

**Comment by vikstrous** [comment]
Seems to work. I tried a 1.12.3 client against a server from this PR:

```
docker exec -it dind docker info
Containers: 0
 Running: 0
 Paused: 0
 Stopped: 0
Images: 0
Server Version: 1.13.0-dev
Storage Driver: vfs
Logging Driver: json-file
Cgroup Driver: cgroupfs
Plugins:
 Volume: local
 Network: bridge host macvlan null overlay
Swarm: inactive
Runtimes: runc
Default Runtime: runc
Security Options: apparmor seccomp
Kernel Version: 4.8.3
Operating System: Alpine Linux v3.4 (containerized)
OSType: linux
Architecture: x86_64
CPUs: 4
Total Memory: 15.53 GiB
Name: 886dba46f468
ID: 7RTU:XCZS:Q7IE:XYH2:Q6WR:L65U:2K6G:IPLS:H7FJ:MC4Y:XBCU:L4L6
Docker Root Dir: /var/lib/docker
Debug Mode (client): false
Debug Mode (server): true
 File Descriptors: 14
 Goroutines: 22
 System Time: 2016-11-07T22:58:25.474393926Z
 EventsListeners: 0
Registry: https://index.docker.io/v1/
WARNING: No cpu cfs quota support
WARNING: No cpu cfs period support
WARNING: bridge-nf-call-iptables is disabled
WARNING: bridge-nf-call-ip6tables is disabled
Insecure Registries:
 127.0.0.0/8
```

**Comment by justincormack** [comment]
LGTM

File: client/client.go (hunk 1 above)
Comment (crosbymichael) [inline client/client.go]: This is unrelated to the actual change in this PR right?  Its just updating the version to what it is supposed to be.

**Comment by crosbymichael** [comment]
LGTM

You can add a comment or whatever @thaJeztah wants about keeping the types synced in another issue/pr ;)
